sections/home/WhatsHappening.jsx

```diff
@@ -2,15 +2,18 @@ import styled from "styled-components";
 
 import PumpDigest from "../../public/home/pump-digest.png";
 import ImpactReport from "../../public/home/impact-report.png";
-import { CardCarousel, Text } from "../../components";
+import { CardCarousel, Text, Video} from "../../components";
 import { baseTheme } from "../../theme";
 import { media } from "../../utils";
 
+
 export const WhatsHappening = ({ ...props }) => (
   <Wrapper {...props}>
     <Title size={baseTheme.size.h1} bold="true">
       What&apos;s Happening?
     </Title>
+
+
```

Comment:
Just a heads up be careful of these blank lines. They shouldn't be here for deployed branches.